sygnal/apnspushkin.py

```diff
@@ -124,39 +130,87 @@ def __init__(self, name, sygnal, config):
         # without this, aioapns will retry every second forever.
         self.apns_client.pool.max_connection_attempts = 3
 
-    async def _dispatch_request(self, log, span, device, shaved_payload, prio):
+    def _map_event_dispatch_handler(self, n):
```

Comment:
```suggestion
    def _map_event_dispatch_handler(self, event):
```

We generally try to avoid single-character variable names if possible.